lib/CallBackery/qooxdoo/callbackery/source/class/callbackery/ui/plugin/Form.js

```diff
@@ -204,18 +210,22 @@ qx.Class.define("callbackery.ui.plugin.Form", {
             }
             rpc.callAsync(function(data,exc){
                 if (!exc){
-                    var statusData = {};
-                    that._cfg.form.forEach(function(item){
-                        if (item.set && item.set.readOnly && data){
-                            statusData[item.key] = data[item.key];
-                        }
-                    });
-                    that._form.setData(statusData,true);
+                    if (that._form) {
```

Comment:
having the same structure everywhere ... would be nice ... 

```
if (!this._form) return;
```
righ at the top